### DISCUSSION TITLE
Release

### DISCUSSION THREAD

**Discussion by phillip-kruger** [smallrye/smallrye-open-api#861] [General]
I think we should release 2.1.8 ?
@MikeEdgar w.d.y.t ?

**Comment by MikeEdgar** [comment]
Sounds good to me!

**Comment by codespearhead** [comment]
@phillip-kruger This seems to be a solved discussion. [Can you close it](https://docs.github.com/en/discussions/managing-discussions-for-your-community/managing-discussions#closing-a-discussion) so that it doesn't show up [in the discussion tab](https://github.com/smallrye/smallrye-open-api/discussions) as an unanswered?
